src/backend/services/workflow_service.py

```
@@ -457,26 +629,23 @@ def run_agentic_workflow(natural_language_query: str, model_provider: str, model
+                if llm_monitor is not None:
+                    unified_metrics.llm_cleaning_stats = llm_monitor.get_numeric_stats()
+
                 collector = get_workflow_metrics_collector()
                 collector.record_workflow(unified_metrics)
-                
-                # 6. Cleanup temp file
-                temp_storage.delete_temp_file(workflow_id)
+
+                _safe_delete_temp_metrics(workflow_id)
                 
                 logging.info(f"✅ Unified metrics recorded successfully")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)